lib/spitfire.ex

```diff
@@ -140,6 +140,21 @@ defmodule Spitfire do
     end
   end
 
+  def parse_with_comments(code, opts \\ []) do
```

Comment:
Please add a comment with the link to Elixir's license here since I think the code was mostly copy pasted. Permalink to the current version.